docs/trestle_author.md

```diff
@@ -18,347 +27,573 @@ While trestle provides editing support for OSCAL there is an unfortunate truth t
 
 The markdown centric workflows allow transition path where capability is [being developed](https://github.com/IBM/compliance-trestle/issues/555)
 
-## Getting Started
+</details>
 
-To work with governed documents, you would need to initialize the Trestle workspace and create your first governed document and template. To do so please follow this [tutorial](./tutorials/work_with_authoring_versions.md).
+<details>
+  <summary><b>2. Setting up the Trestle workspace</b></summary>
 
-## Governance mechanisms
+## Setting up the Trestle workspace
 
-### Markdown structural enforcement
+To work with governed documents, you would need to initialize the Trestle workspace and create your first governed document
+and template.
 
-In order for trestle to enforce structure an approach has been taken for how to template markdown documents. There are two mechanism that are enforced:
+To create a trestle workspace that will contain all your templates and governed documents:
 
-1. Enforcing a heading structure within the markdown document:
-1. Enforcing a structured header within the markdown document either by using yaml headers or a designated heading.
+```bash
+mkdir my_workspace
+cd my_workspace
+trestle init
+>>> Initialized trestle project successfully in [user_path]/my_workspace
+```
 
-For enforcing the heading structure the mechanism used is the following:
+The templates will be located in the `/my_workspace/.trestle/author/`. Please note that if you use MacOS or Linux distribution, by default `.trestle` will be hidden. You will need to reconfigure your system to show hidden folders if you want to see them listed.
 
-Markdown headings `As an example` are considered to be nested based on the heading level (e.g. ` heading` is below ` top level heading`).
+</details>
+
+<details>
+  <summary><b>3. Authoring of Documents</b></summary>
+
+- <details>
+  <summary><b>Documents Structural Enforcement</b></summary>
+
+  ## Markdown structural enforcement
+
+  Trestle templating enforces the documents to follow a specific structure. There are two ways in which structure is enforced in a document against template:
+
+  1. Enforcing a header (metadata) structure within the markdown document.
+  1. Enforcing a heading structure within the markdown document.
```

Comment:
header and heading is confusing.  Should you refer to "header" as "yaml header" for clarity?

Reply:
Changed to YAML header